src/tabs/GetStartedEndUser.tab

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomTab xmlns="http://soap.sforce.com/2006/04/metadata">
+    <description>NPSP Get Started Page</description>
+    <flexiPage>GetStartedEndUser</flexiPage>
+    <label>Get Started with NPSP</label>
```

Comment:
@smarovich This new tab has the identical label "Get Started with NPSP" to the [admin tab](https://github.com/SalesforceFoundation/NPSP/blob/master/src/tabs/GetStarted.tab). That will be confusing to admins who will have visibility to both tabs. There would be no way for them to tell the two tabs apart without opening each. Even the tab icon is the same.

Reply:
@diego-caballero @force2b Ali, Anne, Claire and I agreed that the End User page would be "Get Started with NPSP" and the admin page title would change to "Get Started with NPSP - Admin".

Reply:
I think we decided to use this icon for the tab: https://files.slack.com/files-pri/T7T5PNK3P-F01JG2Z9ZNK/download/custom_component_task.svg